wurst/_handles/Unit.wurst

```diff
@@ -327,7 +327,7 @@ public function unit.isPaused() returns boolean
 	return IsUnitPaused(this)
 
 public function unit.pause()
-	PauseUnit(this, true)
+	BlzPauseUnitEx(this, true)
```

Comment:
Are these identical? Unless we can guarantee that, I wouldn't change it yet.

Reply:
Yeah, I already assumed the same, was talking about it with Donach, and we will do some tests on it. For our project we have some selfmade "AdvancedPause" which is using some exploiting combination of locust and invul while pausing the unit, and as far as I understood from blizz talk, the BlzPauseUnitEx should fix the flaws with old PauseUnit function. I would say it should not replace the normal PauseUnit. We could make it as second pause function `unit.pauseEx()` so it does not change the behaviour of current Maps using it.

Reply:
Replaced and added new "Ex" functions